frontend/components/elements/RawComponent.tsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+// tslint:disable:react-no-dangerous-html
+
+export const RawComponent: React.SFC<{ html: string }> = ({ html }) => (
```

Comment:
Does this wrapper help? I kind of like the ugliness of having to call dangerouslySetInnerHTML everywhere we need it. And the wrapper doesn't do anything beyond that.

Reply:
Moving it to `TextBlockElement` so that `ArticleRenderer` doesn't contain display logic.